apps/publish/content/common.py

```diff
@@ -94,7 +99,7 @@ class BasePublishService(BaseService):
     package_service = PackageService()
 
     def on_update(self, updates, original):
-        self._refresh_associated_items(original)
+        self._publish_associated_items(original)
```

Comment:
I think here it should still do the refresh, so there should be both refresh and publish methods, and from publish_associated_items you can call refresh_associated_items if needed

Reply:
Initially I implemented it as 2 methods but I wanted to avoid reading the associated item from the database 2 times. I think the method is quite clear as it is now, will update the description.